src/main/java/com/google/cloud/tools/gradle/appengine/core/AppEngineCorePluginConfiguration.java

```diff
@@ -293,7 +293,6 @@ private void createDeployAllTask() {
 
               project.afterEvaluate(
                   project -> {
-                    deployAllTask.setDeployConfig(new DeployExtension(deployExtension));
```

Comment:
can you throw a comment in here saying that it's configured in the individual plugins?